invest/templates/invest/high-potential-opportunity-detail.html

```diff
@@ -0,0 +1,231 @@
+                </ol>
+            </div>
+            <div class="column-two-thirds">
+                <img src="{{ page.opportunity_list_image.url }}" />
+
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="proposition-two">
+    <div class="container">
+        <div class="grid-row">
+            <div class="column-one-half">
+                <div class="proposition-two-container">
+                    {{ page.proposition_two|add_export_elements_classes }}
+                </div>
+                <ul class="top-level-list">
+                    <li>{{ page.proposition_two_list_item_one|add_export_elements_classes }}</li>
+                    <li>{{ page.proposition_two_list_item_two|add_export_elements_classes }}</li> 
+                    <li>{{ page.proposition_two_list_item_three|add_export_elements_classes }}</li> 
+                </Ul>
+            </div>
+            <div class="column-one-half">
+                {% if page.proposition_two_video %}
+                    {{ page.proposition_two_video }}
+                {% else %}
+                    <img src="{{ page.proposition_two_image.url }}" />
+                {% endif %}
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="competative-advantages">
+    <div class="container">
+        <div class="grid-row">
+            <h2 class="heading-large">{{ page.competative_advantages_title }}</h2>
+            <div class="column-one-third">
+                <img src="{{ page.competative_advantages_list_item_one_icon.url }}" alt="" />
+                <div>
+                    {{ page.competative_advantages_list_item_one|add_export_elements_classes }}
+                    <div class="clearfix"></div>
+                </div>
+            </div>
+            <div class="column-one-third">
+                <img src="{{ page.competative_advantages_list_item_two_icon.url }}" alt="" />
+                <div>
+                    {{ page.competative_advantages_list_item_two|add_export_elements_classes }}
+                </div>
+            </div>
+            <div class="column-one-third">
+                <img src="{{ page.competative_advantages_list_item_three_icon.url }}" alt="" />
+                <div>
+                    {{ page.competative_advantages_list_item_three|add_export_elements_classes }}
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="testimonial">
+    <div class="container">
+        <div class="grid-row">
+            {{ page.testimonial|add_export_elements_classes }}
+        </div>
+    </div>
+</section>
+
+<section id="company-list">
+    <div class="container">
+        <div class="grid-row">
+            <div>
+                {{ page.companies_list_text|add_export_elements_classes }}
+            </div>
+            <ul>
+                <li class="company-list-first-line">
```

Comment:
just looking at mobile design now. these should be in a grid that collapses to 2 columns on mobile. you can use our shiny new responsive grid classes for that :) https://directory-components-dev.herokuapp.com/responsive-grid/

Reply:
I've not implemented mobile yet. that will be a follow up PR. this is behind a feature flag so it will do no harm, and the PR is getting laaaarge :)

Reply:
okay that is cool 👍